types/jss/index.d.ts

```diff
@@ -17,13 +23,12 @@ export interface Rule {
 	prop(key: string, value: any): this;
 	toJSON(): string;
 }
-export interface StyleSheet<T> {
+
+export interface StyleSheet<Name extends string = any> {
```

Comment:
This appears to be the type of the names of the rules; can we make it `RuleName`?

Reply:
@appsforartists You're talking about `StyleSheet`, right?  If so, that doesn't make sense on at least three counts.  It's inconsistent with JSS per se, which used `createStyleSheet`, and uses `StyleSheet` consistently in the docs.  It's not a singular rule, it's a collection of rules by name - like a CSS stylesheet.  It breaks all current usage.  (What am I missing!!!)

Reply:
This is just a guess, but maybe @appsforartists just wants the name of `Name` changed to `RuleName`? I think it’s a comment on the type parameter, anyway.

Reply:
Fixed